Src/CSharpier/Program.cs

```diff
@@ -67,14 +67,16 @@ CancellationToken cancellationToken
                 TabWidth = configurationFileOptions.TabWidth,
                 UseTabs = configurationFileOptions.UseTabs,
                 Width = configurationFileOptions.PrintWidth,
-                EndOfLine = configurationFileOptions.EndOfLine == "lf"
-                    ? "\n"
-                    : configurationFileOptions.EndOfLine == "crlf"
-                            ? "\r\n"
-                            : throw new Exception(
-                                    "Unhandled value from EndOfLine options "
-                                    + configurationFileOptions.EndOfLine
-                                )
+                EndOfLine = configurationFileOptions.EndOfLine == "auto"
+                    ? EndOfLine.Auto
+                    : configurationFileOptions.EndOfLine == "lf"
```

Comment:
This will probably look much better with pattern matching (`match {}` statement) than nested ternary operators.